sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/ShowTablesExec.scala

```diff
@@ -38,12 +38,14 @@ case class ShowTablesExec(
     val rows = new ArrayBuffer[InternalRow]()
 
     val tables = catalog.listTables(namespace.toArray)
+    // how to make sure the table is matched here? What is pattern and how to use it?
```

Comment:
@cloud-fan

Reply:
You can look for Hive docs about the syntax of the pattern string. `*` matches everything.